src/util-coredump-config.c

```diff
@@ -24,7 +24,6 @@
  */
 
 #include "suricata-common.h"
-#define _FILE_OFFSET_BITS 64
```

Comment:
this is used by system headers, but I'm not sure it's in the correct location here (or if it needed at all)

Reply:
And CI is green...